docs/format.rst

```diff
@@ -12,20 +12,82 @@ A `Lance Dataset` is organized in a directory.
         data/*.lance  -- Data directory
         latest.manifest -- The manifest file for the latest version.
         _versions/*.manifest -- Manifest file for each dataset version.
+        _fragment_manifests/*.lance -- Lance files containing data file metadata.
```

Comment:
*.lance has the same layout as the data file?

Reply:
What is the "unit" of the fragment manifest files here? Should it be a group of fragments, to consolidate the level of files exist in the global manifest and # of manifest files?